posthog/migrations/0110_plugin_storage.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Django 3.0.11 on 2021-01-05 13:46
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("posthog", "0109_fix_retention_filters"),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="PluginStorage",
+            fields=[
+                ("id", models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name="ID")),
+                ("key", models.CharField(max_length=200)),
+                ("value", models.TextField(blank=True, null=True)),
+                (
+                    "plugin_config",
+                    models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to="posthog.PluginConfig"),
+                ),
+                ("team", models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to="posthog.Team")),
+            ],
+        ),
+        migrations.AddIndex(
+            model_name="pluginstorage",
+            index=models.Index(fields=["team_id", "plugin_config_id", "key"], name="posthog_plu_team_id_d6b966_idx"),
```

Comment:
Let's make this a unique index if the semantics allow? This way it's enforced to be consistent and you can use postgres UPSERT for updating data :)

Reply:
sure!